python/opentrons_ot3_firmware/messages/payloads.py

```diff
@@ -139,3 +139,12 @@ class ReadMotorDriverRegisterResponsePayload(utils.BinarySerializable):
 
     reg_addr: utils.UInt8Field
     data: utils.UInt32Field
+
+
+@dataclass
+class ReadPresenceSensingVoltageResponsePayload(utils.BinarySerializable):
+    """Read head presence sensing voltage response payload."""
+
+    z_motor: utils.UInt16Field
```

Comment:
should probably note the unit here - mV? adc least-significant-bits?

Reply:
@sotaoverride ?

Reply:
let me confirm the bits about the units here

Reply:
going off of this sheet[ ](https://docs.google.com/spreadsheets/d/1DMPWo-VjyG0ozShu76oXK-0zwqgU-sOrPBd-KJ6WkTI/edit#gid=785336481) utils.UInt16Field should work fine

Reply:
And what are the units?

Reply:
ADC readings were in mV, this sheet shows the max.min values[ ](https://docs.google.com/spreadsheets/d/1DMPWo-VjyG0ozShu76oXK-0zwqgU-sOrPBd-KJ6WkTI/edit#gid=785336481)

Reply:
https://docs.google.com/spreadsheets/d/1DMPWo-VjyG0ozShu76oXK-0zwqgU-sOrPBd-KJ6WkTI/edit#gid=785336481